src/styles/blocks/header.scss

```diff
@@ -0,0 +1,94 @@
+.header {
+  background-image: url(../images/bitmaps/cover.png);
```

Comment:
Why do you need so a huge image for bg? PNG is for icons or illustrations, but not for photos. Use JPG instead.